### PR TITLE
Make the modal ignore NavigationDuplicated error when updating the query string

### PR DESCRIPTION
**Pull request by SebinSong**
related to #686 .

according to the discussion on this thread(https://github.com/vuejs/vue-router/issues/2872#issuecomment-519073998), NavigationDuplicated error that is thrown when updating a query string is an expected behavior and is safe to ignore.

### CODE REVIEW

**Review by taoeffect — COMMENTED** [review]
Nice, very clean. 👍 

Interestingly, immediately upon creating a new user account via the SignUp modal and then being navigated to http://localhost:8000/app/, I saw this in the console:

```
{…}
​
_name: "NavigationDuplicated"
​
message: "Navigating to current location (\"/\") is not allowed"
​
name: "NavigationDuplicated"
​
stack: "NavigationDuplicated@http://localhost:8000/assets/js/main.js:2073:20\nconfirmTransition@http://localhost:8000/assets/js/main.js:2189:24\ntransitionTo@http://localhost:8000/assets/js/main.js:2133:14\npush@http://localhost:8000/assets/js/main.js:2464:16\npush/<@http://localhost:8000/assets/js/main.js:2878:28\npush@http://localhost:8000/assets/js/main.js:2877:18\nupdateUrl@http://localhost:8000/assets/js/main.js:19399:28\nunloadModal@http://localhost:8000/assets/js/main.js:19422:18\ncreated/<@http://localhost:8000/assets/js/main.js:19329:69\nokTurtlesEventsEmit@http://localhost:8000/assets/js/vue.common-d0684a3c.js:103:21\nsbp@http://localhost:8000/assets/js/events-641a5b8d.js:32:35\n$route@http://localhost:8000/assets/js/main.js:19365:20\nrun@http://localhost:8000/assets/js/translations-e19a2208.js:4567:25\nflushSchedulerQueue@http://localhost:8000/assets/js/translations-e19a2208.js:4311:19\nnextTick/<@http://localhost:8000/assets/js/translations-e19a2208.js:1995:18\nflushCallbacks@http://localhost:8000/assets/js/translations-e19a2208.js:1921:20\n"
```

And then after clicking the "Awesome" button after group creation (being transferred to the dashboard) I saw:

```
_name: "NavigationDuplicated"
​
message: "Navigating to current location (\"/dashboard\") is not allowed"
​
name: "NavigationDuplicated"
​
stack: "NavigationDuplicated@http://localhost:8000/assets/js/main.js:2073:20\nconfirmTransition@http://localhost:8000/assets/js/main.js:2189:24\ntransitionTo@http://localhost:8000/assets/js/main.js:2133:14\npush@http://localhost:8000/assets/js/main.js:2464:16\npush/<@http://localhost:8000/assets/js/main.js:2878:28\npush@http://localhost:8000/assets/js/main.js:2877:18\nupdateUrl@http://localhost:8000/assets/js/main.js:19399:28\nunloadModal@http://localhost:8000/assets/js/main.js:19422:18\ncreated/<@http://localhost:8000/assets/js/main.js:19329:69\nokTurtlesEventsEmit@http://localhost:8000/assets/js/vue.common-d0684a3c.js:103:21\nsbp@http://localhost:8000/assets/js/events-641a5b8d.js:32:35\n$route@http://localhost:8000/assets/js/main.js:19365:20\nrun@http://localhost:8000/assets/js/translations-e19a2208.js:4567:25\nflushSchedulerQueue@http://localhost:8000/assets/js/translations-e19a2208.js:4311:19\nnextTick/<@http://localhost:8000/assets/js/translations-e19a2208.js:1995:18\nflushCallbacks@http://localhost:8000/assets/js/translations-e19a2208.js:1921:20\n"
```

I can approve this PR, as it doesn't claim to fix the issue completely and is a very clean PR. Or I can let you continue working on it before merging. Your call, let me know!

**Comment by SebinSong** [comment]
@taoeffect I would love to work on that error as well. Thanks for letting me know!

**Comment by SebinSong** [comment]
@taoeffect I just tried both signup and group creation processes on both chrome and firefox and none of those errors showed up in my case.

**Comment by taoeffect** [comment]
@SebinSong Hmm. You're right, I just tried again and they're gone. My guess is I must have encountered a browser caching issue. I must have forgotten to explicitly refresh the page after starting Firefox Dev Edition. OK, will merge this!

**Review by taoeffect — APPROVED** [review]
Appears to close #686. We can re-open if any others show up again. Great work @SebinSong! 👍

**Comment by SebinSong** [comment]
@taoeffect Thanks! :)

**Pull request merged**
